datumaro/plugins/pointcloud_format/extractor.py

```diff
@@ -0,0 +1,131 @@
+import json
+import os
+
+from collections import OrderedDict
+import os.path as osp
+
+
+from datumaro.components.extractor import (SourceExtractor, DatasetItem,
+    AnnotationType,Cuboid,
+    LabelCategories, Importer
+)
+
+from datumaro.util.image import Image
+
+from .format import PointCloudPath
+
+
+class PointCloudExtractor(SourceExtractor):
+    _SUPPORTED_SHAPES = "cuboid"
+    mapping = {}
+    meta = {}
```

Comment:
I don't think these variables should be global

Reply:
Resolved